workflows/prognostic_c48_run/runtime/tendency_prescriber.py

```diff
@@ -0,0 +1,99 @@
+import dataclasses
+import logging
+from typing import Hashable, Mapping, MutableMapping, Set
+
+import cftime
+import fsspec
+import xarray as xr
+import zarr
+
+import fv3gfs.util
+from runtime.monitor import Monitor
+from runtime.types import Diagnostics, Step
+from runtime.derived_state import DerivedFV3State
+from runtime.utils import quantity_state_to_ds, ds_to_quantity_state
+
+QuantityState = MutableMapping[Hashable, fv3gfs.util.Quantity]
+
+logger = logging.getLogger(__name__)
+
+
+@dataclasses.dataclass
+class TendencyPrescriberConfig:
+    """Configuration for overriding tendencies from a step.
+    
+    Attributes:
+        url: path to dataset containing tendencies.
+        variables: mapping from state name to name of corresponding tendency in
+            provided dataset. For example: {"air_temperature": "fine_res_Q1"}.
+    """
+
+    url: str
+    variables: Mapping[str, str]
+
+
+@dataclasses.dataclass
+class TendencyPrescriberAdapter:
```

Comment:
Is there some API being "adapted"? The word adapter refers to the wrapping the ML model (which has its own class hierarchy) with a suitable class with the API expected by the loop.py. In this case there is no underlying class to adapt, so I would simply call this "TendencyPrescriber". 
```suggestion
class TendencyPrescriber:
```